### PR TITLE
exiv2: 0.26.x -> 0.27.x

### PR DESCRIPTION
**Pull request by periklis**
###### Motivation for this change
fix for #50369

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [x] other Linux distributions (nix-docker)
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by Mic92** [comment]
@GrahamcOfBorg build exiv2

**Comment by Mic92** [comment]
@GrahamcOfBorg build python36Packages.py3exiv2 gegl_0_4

**Comment by vcunat** [comment]
They've released RC2 already.  There are some new Coverity fixes in the log, so that's possibly also relevant for security.

**Comment by vcunat** [comment]
The new build system also installs very many new executables.  Maybe there were some other caveats; I've forgotten since the last time I've tried pre-0.27.

**Comment by ckauhaus** [comment]
@periklis I'd rather put this update on hold until 0.27 has been released.

**Comment by vcunat** [comment]
Expected dates [have been announced](http://dev.exiv2.org/boards/3/topics/2830):

> GM will be v0.27.0.0 Scheduled: 28 December 2018
> RC3 will be v0.27.0.3 Scheduled: 07 November 2018

But I guess RC3 should be 07 _December_, given that the announcement was on 16 November :-)

**Comment by vcunat** [comment]
Well, the real problem with 0.27 is that a large fraction of direct dependents won't build with it (yet).

**Comment by jtojnar** [comment]
gexiv2 0.10.10 supporting exiv2 0.27 was released https://gitlab.gnome.org/GNOME/gexiv2/blob/gexiv2-0.10.10/NEWS

**Comment by vcunat** [comment]
Let me link [ghtumb patch](https://gitlab.gnome.org/GNOME/gthumb/issues/30), too.  ATM it still seems early to update, but feel free to try finding/producing patches for most of our packages.

File: pkgs/desktops/gnome-3/misc/gexiv2/default.nix [hunk 1]
```diff
@@ -12,6 +14,14 @@ stdenv.mkDerivation rec {
     sha256 = "1vf0zv92p9hybdhn7zx53h3ia53ph97a21xz8rfk877xlr5261l8";
   };
 
+  patches = [
+    (fetchpatch { # should be included in the following release
```
Comment (jtojnar) [inline pkgs/desktops/gnome-3/misc/gexiv2/default.nix]: gexiv2 0.10.10 is in master now.

**Review by jtojnar — CHANGES_REQUESTED** [review]

**Comment by aanderse** [comment]
I happened to notice 0.27.1 was release recently: https://www.exiv2.org/whatsnew.html

**Pull request closed without merging**